docker-compose.yml

```diff
@@ -1,6 +1,17 @@
 version: '3'
 
 services:
+    web:
```

Comment:
the way you could put this in the formula is to run this container in that `docker-compose.yml`, but front it with the host-based nginx forwarding everything to port `8000` and providing HTTPS. Although the proxies sequence gets longer and longer.